parquet-column/src/main/java/org/apache/parquet/schema/LogicalTypeAnnotation.java

```diff
@@ -147,6 +151,22 @@ protected LogicalTypeAnnotation fromString(List<String> params) {
         return float16Type();
       }
     },
+    GEOMETRY {
```

Comment:
@zhangfengcdt, is it possible to break up this PR and get just the logical type annotations and translation in separately? This is quite large and would be easier to review in parts. And getting the annotations out first allows us to start building test files to share.

Reply:
Awesome, thank you!